server/subscribe.go

```diff
@@ -57,15 +71,15 @@ func (s *ChannelSubscriptions) remove(sub *ChannelSubscription) {
 
 	s.IdByChannelId[sub.ChannelId] = remove(s.IdByChannelId[sub.ChannelId], sub.Id)
 
-	for _, event := range sub.Filters["events"] {
+	for _, event := range sub.Filters.Event {
```

Comment:
Well, you'll need an `Subtract(a Set, elems ...string) Set` too to make this compact